File: 대표값.py
import math
c = []
add = 0
mini = 0
N = int(input())
if 5 <= N and N <= 100 :
    a = list(map(int,input().split()))
    if len(a) == N :
        for i in range(len(a)):
            add = a[i] + add
        add = round(add/len(a))
        for i in range(len(a)):
            c.append(abs(add - a[i]))
        for i in range(len(c)):
            if c[i] < c[mini]:
                mini = i
            elif c[i] == c[mini]:
                if a[i] > a[mini]:
                    mini = i
                elif a[i] == a[mini]:
                    if i < mini:
                        mini = i
        print("%d %d" % (add, mini+1))
    else :
        print("오류!")
else : 
    print("오류!")

File: 대표값.java
import java.util.*;

public class 대표값 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int add = 0;
		int mini = 0;
		int N = sc.nextInt();
		int[] a = new int[N];
		int[] c = new int[N];
		if(5 <= N && N <= 100) {
			for(int i = 0;i<a.length;i++) {
				a[i] = sc.nextInt();
				add = a[i] + add;
			}
			if(a.length == N) {
				add = (add+5)/10*10;
				add = (add/a.length);
				for(int i = 0;i<a.length;i++) {
					c[i] = Math.abs(add-a[i]);
				}
				for(int i = 0;i<c.length;i++) {
					if(c[i] < c[mini]) {
						mini = i;
					}
					else if(c[i] == c[mini]) {
						if(a[i] > a[mini]) {
							mini = i;
						}
						else if(a[i] == a[mini]) {
							if (i < mini) {
								mini = i;
							}
						}
					}
				}
			}
			
		}
		System.out.printf("%d %d", add, mini+1);	
	}
}